src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -298,17 +298,42 @@ public void Initialize()
             }
         }
 
-        public async Task PutEditVersioningCommandAsync(JsonOperationContext context, string databaseName, BlittableJsonReaderObject val)
+        public async Task<long> ModifyDatabaseExpirationBundle(TransactionOperationContext context, string name, BlittableJsonReaderObject configurationJson)
```

Comment:
This is not a hot-path and now you are relying on everyone that will change that to remember that somewhere you need to apply additional change. Not worth having this kind of potential bug-spot.